mkdocs/docs/HPC/troubleshooting.md

```diff
@@ -479,4 +479,27 @@ The following have been reloaded with a version change:
 This might result in the same problems as mentioned above. When swapping
 to a different cluster, you can run `module purge` to unload all modules
 to avoid problems (see [Purging all modules](../running_batch_jobs/#purging-all-modules)).
+
+### Multi-job submissions on a non-default cluster
+
+When using multi-job submissions, it could be possible you get following error when you try to run it on a non-default cluster.
+
+<pre><code>$ <b> wsub</b>
+/apps/gent/RHEL8/skylake-ib/software/worker/1.6.13-iimpi-2021b/bin/wsub: line 27: 2152510 <b>Illegal instruction</b>     (core dumped) ${PERL} ${DIR}/../lib/wsub.pl "$@"
```

Comment:
Avoid using a specific version (or even OS/CPU arch), since this will go out of data in no time:

```suggestion
/apps/gent/.../.../software/worker/.../bin/wsub: line 27: 2152510 <b>Illegal instruction</b>     (core dumped) ${PERL} ${DIR}/../lib/wsub.pl "$@"